src/ocean/LibFeatures.d

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+
+    Meta module defining constants that match ocean feature releases allowing
+    downstream libraries to conditionally implement functionality relying on
+    later ocean versions.
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.LibFeatures;
+
+const has_features_2_5 = true;
+const has_features_2_4 = true;
```

Comment:
Remember that "somebody" is library when it comes to writing the condition, but it will be application which checks it. Applications are not supposed/allowed to use non-supported minor releases while libraries often have to.

Thus in your case library will be on version 2.3 but implementing a feature needing 2.4 under relevant `static if`. However application using that library will use 2.5 at least, which will have all flags for v2.x defined, including 2.5 one.

Alternative would be to check for version 2.5 in that library despite the fact needed feature actually came in 2.4 - I don't mind that too much, but such situation seems much more confusing to me when someone will read that library code.